input4MIPs_source_id_zn_2.json

```diff
@@ -0,0 +1,23 @@
+{
+    "source_id":{
+        "CR-CMIP-0-2-0":{
+            "data_producer_extras":{},
+            "data_producer_required":{
+                "source_id":{
+                    "CR-CMIP-0-2-0":{
+                        "activity_id":"input4MIPs",
+                        "contact":"zebedee.nicholls@climate-resource.com;malte.meinshausen@climate-resource.com",
+                        "further_info_url":"TBD",
+                        "institution":"Climate Resource",
+                        "institution_id":"CR",
+                        "license":"CMIP greenhouse gas concentration data produced by Climate Resource (CR) is licensed under a Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). Consult https://pcmdi.llnl.gov/CMIP6Plus/TermsOfUse for terms of use governing CMIP6Plus and input4MIPs output, including citation requirements and proper acknowledgment. Further information about this data, can be found at TBC. The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.",
+                        "mip_era":"CMIP6Plus",
+                        "source_version":"0.2.0",
+                        "version":"0.2.0"
+                    }
+                }
+            },
+            "esgf":{}
```

Comment:
If we went with this option, the idea would be that the ESGF relevant keys can be added over time. This means that you get changes in this file which aren't actually to do with the source ID information from the data producer changing, which feels wrong, but maybe it is the least worst compromise.

Reply:
I've got a new variant of this, will merge this into what I have and we can iterate

Reply:
All sounds like it's going in the right direction. How long until the upstream changes are implemented and we can start using them here?